addons/dialog/functions/fnc_gui_owners.sqf

```diff
@@ -0,0 +1,234 @@
+#include "script_component.hpp"
+/*
+ * Author: mharis001
+ * Initializes the OWNERS content control.
+ *
+ * Arguments:
+ * 0: Controls Group <CONTROL>
+ * 1: Row Index <NUMBER>
+ * 2: Current Value <ARRAY>
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * [CONTROL, 0, [1, [west], [], []]] call zen_dialog_fnc_gui_owners
+ *
+ * Public: No
+ */
+
+params ["_controlsGroup", "_rowIndex", "_currentValue"];
+_currentValue params ["_tab", "_sides", "_groups", "_players"];
+
+_controlsGroup setVariable [QGVAR(params), [_groups, _players]];
+
+private _fnc_onTabClick = {
+    params ["_ctrlButton"];
+    (_ctrlButton getVariable QGVAR(params)) params ["_currentValue"];
+
+    private _controlsGroup = ctrlParentControlsGroup _ctrlButton;
+    private _selectedTab = IDCS_ROW_OWNERS_BTNS find ctrlIDC _ctrlButton;
+
+    {
+        private _ctrlButton = _controlsGroup controlsGroupCtrl _x;
+        _ctrlButton ctrlEnable (_forEachIndex != _selectedTab);
+    } forEach IDCS_ROW_OWNERS_BTNS;
+
+    {
+        private _ctrlTab = _controlsGroup controlsGroupCtrl _x;
+        _ctrlTab ctrlShow (_forEachIndex == _selectedTab);
+    } forEach IDCS_ROW_OWNERS_TABS;
+
+    ctrlSetFocus (_controlsGroup controlsGroupCtrl (IDC_ROW_OWNERS_TABS select _selectedTab));
+
+    _currentValue set [0, _selectedTab + 1]; // +1 since one-based index
```

Comment:
Why `+1 since one-based index`? It seems counter-intuitive of a comment. If it was one-based then you would subtract one not add one.

Reply:
More so that the index used by the control type itself (1 - Sides, 2 - Groups, 3 - Players). I wasn't sure on this but it does enable:

```sqf
_values params ["_owners"];
_owners = _owners select (_owners select 0);
```

Could change to 0-2 or move to last argument in value info.